javascript-conditionals/main.js

```diff
@@ -0,0 +1,81 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+const isUnderFive = (num) => {
+  return (num < 5);
+};
+
+const isEven = (num) => {
+  return (num % 2 === 0);
+};
+
+const startsWithJ = (str) => {
+  return (str[0] === 'J');
+};
+
+const isOldEnoughToDrink = (person) => {
+  return (person.age > 20);
+};
+
+const isOldEnoughToDrive = (person) => {
+  return (person.age > 15);
+};
+
+const isOldEnoughToDrinkAndDrive = (person) => {
+  return false;
+};
+
+const categorizeAcidity = (pH) => {
+  if (pH === 7) {
+    return 'neutral';
+  } else if (pH < 7 && pH > 0) {
+    return 'acid';
+  } else if (pH < 14 && pH > 7) {
+    return 'base';
```

Comment:
What happens if `ph` is 14? 14 would still be considered base

Reply:
Thank you! Constantly struggling with edge cases lol